### PR TITLE
add title to all page

### PR DESCRIPTION
**Pull request by phuongpham167**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, current_user.name) %>
 <div class="container-fluid" style="padding-left: 300px; padding-top: 100px;">
```
Comment (nguyenhongphuc2018) [inline app/views/users/show.html.erb:2]: css inline
just move it into your file css

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -18,4 +18,13 @@ def load_rating
       @product.update_avg_point
     end
   end
+
+  def full_title(page_title = '')
```
Comment (nguyenhongphuc2018) [inline app/helpers/application_helper.rb:22]: brackets

File: app/views/admin/categories/edit.html.erb [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Edit category") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/categories/edit.html.erb:1]: try remove brackets and i18n

File: app/views/admin/categories/index.html.erb [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Manage Categories") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/categories/index.html.erb:1]: try remove brackets and i18n

File: app/views/admin/categories/new.html.erb [hunk 5]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Create new category") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/categories/new.html.erb:1]: same as above

File: app/views/admin/categories/show.html.erb [hunk 6]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, @category.name) %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/categories/show.html.erb:1]: same as above

File: app/views/admin/orders/index.html.erb [hunk 7]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Manage orders") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/orders/index.html.erb:1]: same as above

File: app/views/admin/product_temps/index.html.erb [hunk 8]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Manage suggests") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/product_temps/index.html.erb:1]: same as above

File: app/views/admin/products/edit.html.erb [hunk 9]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Edit product") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/products/edit.html.erb:1]: same as above

File: app/views/admin/products/index.html.erb [hunk 10]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Manage Products") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/products/index.html.erb:1]: same as above

File: app/views/admin/products/new.html.erb [hunk 11]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Create new product") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/products/new.html.erb:1]: same as above

File: app/views/admin/users/index.html.erb [hunk 12]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Manage Users") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/users/index.html.erb:1]: same as above

File: app/views/admin/users/new.html.erb [hunk 13]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Create new user") %>
```
Comment (nguyenhongphuc2018) [inline app/views/admin/users/new.html.erb:1]: same as above

File: app/helpers/application_helper.rb [hunk 14]
```diff
@@ -18,4 +18,13 @@ def load_rating
...
+    base_title = "Tasty bunny"
```
Comment (Dat1606) [inline app/helpers/application_helper.rb:23]: I18n

File: app/views/carts/index.html.erb [hunk 15]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Cart") %>
```
Comment (nguyenhongphuc2018) [inline app/views/carts/index.html.erb:1]: same as above

File: app/views/likes/index.html.erb [hunk 16]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Your favorite") %>
```
Comment (nguyenhongphuc2018) [inline app/views/likes/index.html.erb:1]: same as above

File: app/views/orders/index.html.erb [hunk 17]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Your orders") %>
```
Comment (nguyenhongphuc2018) [inline app/views/orders/index.html.erb:1]: same as above

File: app/views/orders/new.html.erb [hunk 18]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "New order") %>
```
Comment (nguyenhongphuc2018) [inline app/views/orders/new.html.erb:1]: same as above

File: app/views/orders/show.html.erb [hunk 19]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Order detail") %>
```
Comment (nguyenhongphuc2018) [inline app/views/orders/show.html.erb:1]: same as above

File: app/views/product_temps/index.html.erb [hunk 20]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Your suggest") %>
```
Comment (nguyenhongphuc2018) [inline app/views/product_temps/index.html.erb:1]: same as above

File: app/helpers/application_helper.rb [hunk 21]
```diff
@@ -18,4 +18,13 @@ def load_rating
...
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
+  end
```
Comment (Dat1606) [inline app/helpers/application_helper.rb:29]: làm khác 1 chút so với tut được không chị?

File: app/views/products/index.html.erb [hunk 22]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Products") %>
```
Comment (nguyenhongphuc2018) [inline app/views/products/index.html.erb:1]: same as above

File: app/views/products/show.html.erb [hunk 23]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, @product.name) %>
```
Comment (nguyenhongphuc2018) [inline app/views/products/show.html.erb:1]: same as above

File: app/views/static_pages/contact.html.erb [hunk 24]
```diff
@@ -1,3 +1,4 @@
+<% provide(:title, "Contact") %>
```
Comment (nguyenhongphuc2018) [inline app/views/static_pages/contact.html.erb:1]: same as above

File: app/helpers/application_helper.rb (hunk 21 above)
Comment (phuongpham167) [reply app/helpers/application_helper.rb:29]: có em chị có thể dùng page_title.empty? base_title : page_title + "|" + base_title

**Pull request merged**